src/ui/utils/prefs.ts

```diff
@@ -98,7 +98,3 @@ export const features = new PrefsHelper("devtools.features", {
 });
 
 export type Features = typeof features;
-
-export const asyncStore = asyncStoreHelper("devtools", {
-  replaySessions: ["Json", "replay-sessions", {} as Record<string, ReplaySession>],
-});
```

Comment:
I hope to rewrite this stuff soon but for now, moving this inside of a protected block helps.